supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/MachineConstraints.scala

```diff
@@ -21,6 +23,13 @@ object MachineConstraints {
   private val maxMemoryPerCpu = MemorySize(6.5, MemoryUnit.GB)
   private val memoryFactor = MemorySize(256, MemoryUnit.MB)
 
+  private lazy val log = LoggerFactory.getLogger("MachineConstraints")
+  private lazy val usePredefinedMachineTypes: Boolean = {
+    val predefinedMachineTypes = ConfigFactory.load().getOrElse("google.papiv2-use-v1-style-machine-types", false)
```

Comment:
Sad that:
* You have to reload the config again
* This is a top-level option rather than a per-backend option (or indeed, a per-job option

OTOH, this seems fine for now for testing purposes

Reply:
right this is definitely not how this would work in production 🙂